com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -1,4 +1,5 @@
 using System.Collections.Generic;
+using UnityEditor.VersionControl;
```

Comment:
Was this added by mistake?

Reply:
Yes. I think Rider IDE has auto-added it for some reason and it broke all the tests.